src/sv-pipeline/scripts/vcf_qc/analyze_fams.R

```diff
@@ -125,12 +125,17 @@ getFamDat <- function(dat,proband,father=NA,mother=NA,biallelic=T,nocall.placeho
                    sort=F,by="VID",all=T,suffixes=c(".pro",".mo"))
   }
   #Only retain sites where all three samples are not null genotype (no-call, ./., nocall.placeholder)
-  exclude <- which(sapply(vlist[,c(2,4,6)],
+  exclude <- sapply(vlist[,c(2,4,6)],
                           function(vals){
-    any(as.numeric(vals)==nocall.placeholder)
-  }))
-  if(length(exclude) > 0){
-    vlist <- vlist[-exclude,]
+    which(as.numeric(vals)==nocall.placeholder)
+  })
+
+  exclude_out = c()
+  for(exclude_x in exclude){  exclude_out=c(exclude_out,exclude_x) }
+  exclude_out=unique(exclude_out)
```

Comment:
Can't this block just be written as `exclude_out <- unique(exclude)`?

Reply:
no, this code is to transfer data formats